boot/makebootable.go

```diff
@@ -312,6 +312,25 @@ type makeRunnableOptions struct {
 	AfterDataReset bool
 }
 
+func copyNoSymLink(orig, dst string) error {
+	// if the source path is a symlink, don't copy the symlink, copy the
+	// target file instead of copying the symlink, as the initramfs won't
+	// follow the symlink when it goes to mount the base and kernel snaps by
+	// design as the initramfs should only be using trusted things from
+	// ubuntu-data to boot in run mode
+	if osutil.IsSymlink(orig) {
```

Comment:
Do you want this to be a `while` to be on the safe side?

Reply:
I guess we could have used `filepath.EvalSymlinks()` but it should be fine, might be followup material.